Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -197,508 +186,263 @@
-        "metrics": [
-          {
-            "namespace": "microsoft.documentdb/databaseaccounts",
-            "metric": "microsoft.documentdb/databaseaccounts-Requests-IndexUsage",
-            "aggregation": 1,
-            "chartType": 2,
-            "splitBy": null
-          },
-          {
-            "namespace": "microsoft.documentdb/databaseaccounts",
-            "metric": "microsoft.documentdb/databaseaccounts-Requests-DataUsage",
-            "aggregation": 1
-          },
-          {
-            "resourceId": "{ResourceId}"
-          },
-          {
-            "resourceId": "{ResourceId}"
-          }
         ],
-        "title": "Data and Index Usage",
-        "gridFormatting": []
-      },
-      "conditionalVisibility": {
-        "parameterName": "ResourceId",
-        "comparison": "isNotEqualTo",
-        "value": null
-      },
-      "customWidth": "50",
-      "name": "metric - 13"
-    },
-    {
-      "type": 10,
-      "content": {
-        "chartId": "workbook2d8fc4b8-3c85-44e4-8dd6-b99f7561ca13",
-        "version": "MetricsItem/1.0",
-        "size": 0,
-        "timeContext": {
-          "durationMs": 3600000
-        },
-        "resourceType": "microsoft.documentdb/databaseaccounts",
-        "resourceParameter": "ResourceId",
-        "metrics": [
-          {
-            "namespace": "microsoft.documentdb/databaseaccounts",
-            "metric": "microsoft.documentdb/databaseaccounts-Requests-DocumentCount",
-            "aggregation": 1,
-            "chartType": 2,
-            "splitBy": null
-          },
-          {
-            "namespace": "microsoft.documentdb/databaseaccounts",
-            "metric": "microsoft.documentdb/databaseaccounts-Requests-DocumentQuota",
-            "aggregation": 1
-          },
-          {
-            "resourceId": "{ResourceId}"
-          },
-          {
-            "resourceId": "{ResourceId}"
-          }
-        ],
-        "title": "Document Count and Quota",
-        "gridFormatting": []
-      },
-      "conditionalVisibility": {
-        "parameterName": "ResourceId",
-        "comparison": "isNotEqualTo",
-        "value": null
-      },
-      "customWidth": "50",
-      "name": "metric - 14"
-    },
-    {
-      "type": 1,
-      "content": {
-        "json": "*Data and Index Usage* -  \r\n<br />\r\n<br />"
-      },
-      "conditionalVisibility": {
-        "parameterName": "ResourceId",
-        "comparison": "isNotEqualTo",
-        "value": null
-      },
-      "customWidth": "50",
-      "name": "text - 15"
-    },
-    {
-      "type": 1,
-      "content": {
-        "json": "*Document Count and Quota* -  \r\n<br />\r\n<br />"
-      },
-      "conditionalVisibility": {
-        "parameterName": "ResourceId",
-        "comparison": "isNotEqualTo",
-        "value": null
+        "filters": []
       },
-      "customWidth": "50",
-      "name": "text - 16"
+      "showPin": true,
+      "name": "storage account metrics"
```

Comment:
this step name should be changed to cosmosdb metrics

Reply:
good catch